egs/mgb2/asr/Results.txt

```diff
@@ -0,0 +1,17 @@
+stage 5: Decoding
+best val scores = [0.90567905 0.9056724  0.90525274 0.90449862 0.90426245]
+selected epochs = [36 54 41 22 20]
+average over ['exp/train_trim_sp_pytorch_2_specaug/results/snapshot.ep.36', 'exp/train_trim_sp_pytorch_2_specaug/results/snapshot.ep.54', 'exp/train_trim_sp_pytorch_2_specaug/results/snapshot.ep.41', 'exp/train_trim_sp_pytorch_2_specaug/results/snapshot.ep.22', 'exp/train_trim_sp_pytorch_2_specaug/results/snapshot.ep.20']
+write a CER (or TER) result in exp/train_trim_sp_pytorch_2_specaug/decode_dev_model.val5.avg.best_decode_lm_transformer/result.txt
+| SPKR                                          | # Snt  # Wrd | Corr    Sub    Del    Ins    Err  S.Err |
+| Sum/Avg                                       | 5002  111250 | 88.5    6.0    5.5    1.9   13.4   60.1 |
+write a WER result in exp/train_trim_sp_pytorch_2_specaug/decode_dev_model.val5.avg.best_decode_lm_transformer/result.wrd.txt
+| SPKR                                          | # Snt  # Wrd | Corr    Sub    Del    Ins    Err  S.Err |
+| Sum/Avg                                       | 5002   60166 | 86.5    8.6    5.0    1.1   14.6   59.3 |
+write a CER (or TER) result in exp/train_trim_sp_pytorch_2_specaug/decode_test_model.val5.avg.best_decode_lm_transformer/result.txt
+| SPKR                                          | # Snt  # Wrd | Corr    Sub    Del    Ins    Err  S.Err |
+| Sum/Avg                                       | 5365  119010 | 87.8    6.2    6.1    2.2   14.4   60.8 |
+write a WER result in exp/train_trim_sp_pytorch_2_specaug/decode_test_model.val5.avg.best_decode_lm_transformer/result.wrd.txt
+| SPKR                                          | # Snt  # Wrd | Corr    Sub    Del    Ins    Err  S.Err |
+| Sum/Avg                                       | 5365   64297 | 86.9    8.4    4.7    1.2   14.2   60.1 |
+Finished
```

Comment:
Not sure if we need to report result on overlap data. Non-overlap seems to be good enough to compare. @sw005320 Could you give some suggestion?

Reply:
I think it would be informative if we have both data.

Also, @AmirHussein96, could you change the result file (rename it with `RESULTS.md`) by following the other recipe? (e.g., https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/RESULTS.md#pytorch-large-transformer-with-specaug-4-gpus--transformer-lm-4-gpus)
You can follow 3rd item in https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#how-to-contribute-to-espnet
We also appreciate if you upload your model.

Reply:
> Please fix the CI errors, e.g., https://github.com/espnet/espnet/pull/2289/checks?check_run_id=971493160#step:7:133
> You can also perform the same test locally by following https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#unit-testing


Thank you @sw005320  for all your feedback. I fixed the CI errors, however right now I am getting another error because of my local mgb2 python `scripts local/*.py`  https://github.com/espnet/espnet/pull/2289/checks?check_run_id=993023702. Do I need to fix these?

Reply:
Please fix. You need to install black if you don't have and follows the ci message.

Reply:
Regarding the pretrained models, we will upload them in the second stage, once we finish from a paper we are working on.

Reply:
OK, very cool.
I'll send you our Google drive invitation. Please create an appropriate folder and upload it then.

Reply:
I am having a problem with the line continuation https://github.com/espnet/espnet/pull/2289/checks?check_run_id=994066763 . I tried "\" and """  """ , and although both correct as python code, I can not pass the CI test. Any suggestions?